include/sound/sof/header.h

```diff
@@ -100,6 +101,16 @@
 #define SOF_IPC_STREAM_VORBIS_PARAMS		SOF_CMD_TYPE(0x010)
 #define SOF_IPC_STREAM_VORBIS_FREE		SOF_CMD_TYPE(0x011)
 
+/* probe */
+#define SOF_IPC_PROBE_INIT			SOF_CMD_TYPE(0x001)
+#define SOF_IPC_PROBE_DEINIT			SOF_CMD_TYPE(0x002)
+#define SOF_IPC_PROBE_DMA_SET			SOF_CMD_TYPE(0x003)
+#define SOF_IPC_PROBE_DMA_GET			SOF_CMD_TYPE(0x004)
+#define SOF_IPC_PROBE_DMA_DETACH		SOF_CMD_TYPE(0x005)
+#define SOF_IPC_PROBE_POINT_SET		SOF_CMD_TYPE(0x006)
+#define SOF_IPC_PROBE_POINT_GET		SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PROBE_POINT_REMOVE		SOF_CMD_TYPE(0x008)
```

Comment:
this looks odd, everything is symmetric except for 'remove', there's no 'insert'.

Reply:
Meaning the wording? All the definitions are based on SOF FW. Can speak with Arthur about the naming : )

cAVS uses the following convention:
* probe_point - implies _CONNECT, but due to its dual nature (SET/GET) is left without said suffix
* probe_point_disconnect - explicit
* injection_dma - same as with probe_point, implicit. Here we imply word _ATTACH
* injection_dma_detach - explicit

Reply:
even more confused with the answers... I shouldn't have to learn 5+ years of cAVS conventions to figure out what an IPC means.